### DISCUSSION TITLE
how to update frigate url in Home Assistant add-on?

### DISCUSSION THREAD

**Discussion by Cold-Lemonade** [blakeblackshear/frigate#7328] [Ask A Question] [answered]
The name of the server hosting Frigate has changed on my network. For example, suppose it was frigate_serverA:5000 before and now it is frigate_serverB:5000. How do I update the Frigate add-on in Home Assistant to use the new url?

**Comment by NickM-27** [comment]
If you are running frigate externally then you are not running the addon. I suppose you mean integration and in that case the only way is to remove the integration and then add it again.

**Reply by Cold-Lemonade** [reply]
Sorry. I just realized that I meant integration.

Does that mean I will lose all of my automations that use it?

**Reply by NickM-27** [reply]
No, removing an integration does not delete automations. Just need to make sure the entity ids are the same as before

**Reply by Cold-Lemonade** [reply]
Sorry I am a little confused. Will it delete my frigate-related entities? And then i need to be sure to use the same entity ids as before when I add the integration fresh?

**Reply by NickM-27** [reply, marked as answer]
yes it will delete the entities, and then when you re-add the integration it will add them all back with the default names. If you never edited the names then nothing will have changed and it will continue to work as it did before. 

If you have changed them, you'll want to make note of the changes and reapply them

**Reply by Cold-Lemonade** [reply]
Okay. Thanks. I deleted the Frigate integration and re-installed it. Everything seems to be working okay now.

I really appreciate your help and look forward to the release of Frigate 0.13.
